buildconfig/Jenkins/Conda/build

```diff
@@ -39,6 +40,22 @@ cmake --preset=${CMAKE_PRESET} ${MANTID_DATA_STORE_CMAKE} ${EXTRA_CMAKE_FLAGS} $
 
 # Run the actual builds for the code, unit tests, and system tests.
 cd $WORKSPACE/build
+
+if [[ $ENABLE_COVERITY == true ]]; then
+    ${COVERITY_DIR}/cov-build --dir cov-int cmake --build . $BUILD_OPTIONS
+    tar czvf mantid.tgz cov-int
+    status=$(curl --form token=$COVERITY_TOKEN --form email=mantidproject@gmail.com \
+		  --form file=@mantid.tgz --form version=$GIT_COMMIT \
+		  https://scan.coverity.com/builds?project=mantidproject%2Fmantid)
+    status=$(echo ${status} | sed -e 's/^ *//' -e 's/ *$//')
```

Comment:
mind boggling :space_invader:

Reply:
curl has evolved over the years so it finally gives actual return codes. This is from the dark ages when you had to parse the output of the command. It'd take a little bit to modernize it.